Playmate/Utilities/Helpers.h

```diff
@@ -24,6 +25,11 @@ NS_ASSUME_NONNULL_BEGIN
 + (UIImage *)resizeImage:(UIImage *)image withDimension:(int)dimension;
 + (NSString *)getTimeGivenDurationForSession:(Session *)session;
 
+// For other stuff
```

Comment:
When writing comments leave more meaningful sentences, it'll help you and others understand when reading :P